bdk-ffi/src/lib.rs

```diff
@@ -396,6 +396,12 @@ impl Address {
     fn as_string(&self) -> String {
         self.address.to_string()
     }
+
+    fn from_script(script: Arc<Script>, network: Network) -> Result<Self, BdkError> {
```

Comment:
Because this is a constructor, let's bring it up just below the `new()` method (on line 367).

Reply:
done.